Dockerfile

```diff
@@ -82,7 +82,7 @@ COPY patches ./patches
 # ---------------------------------------------------------
 FROM yarn-base as yarn-deps
 
-RUN --mount=type=cache,target=/usr/local/share/.cache \
+RUN --mount=type=cache,target=/usr/local/share/.cache/yarn \
```

Comment:
This should not be necessary however docker would not run without it.

https://app.circleci.com/pipelines/github/artsy/force/15114/workflows/b7572059-a807-4da2-922b-c6cd72c5e4ef/jobs/111695